frontend/package.json

```diff
@@ -15,35 +15,25 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
-    "@types/react-highlight-words": "^0.16.4",
     "axios": "^0.26.1",
+    "classnames": "^2.3.1",
     "env-cmd": "^10.1.0",
     "js-file-download": "^0.4.12",
-    "material-ui-search-bar": "^1.0.0",
     "react": "^17.0.2",
     "react-canvas-confetti": "^1.3.0",
     "react-dom": "^17.0.2",
     "react-highlight-words": "^0.18.0",
-    "react-horizontal-stacked-bar-chart": "^8.15.2",
-    "react-minimal-pie-chart": "^8.2.0",
     "react-redux": "^7.2.6",
     "react-router-dom": "^6.2.2",
     "react-scripts": "^5.0.1",
     "react-toastify": "^8.2.0",
     "react-tooltip": "^4.2.21",
     "redux": "^4.1.2",
-    "uninstall": "^0.0.0",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start:dev": "env-cmd -f .env.development react-scripts start",
-    "start:prod": "env-cmd -f .env.production react-scripts start",
-    "start:remote": "env-cmd -f .env.remote react-scripts start",
-    "build:testing": "env-cmd -f .env.testing react-scripts build",
+    "start": "env-cmd -f .env.development react-scripts start",
     "build": "env-cmd -f .env.production react-scripts build",
-    "build:remote": "env-cmd -f .env.remote react-scripts build",
-    "build:development": "env-cmd -f .env.development react-scripts build",
-    "build:production.auth": "env-cmd -f .env.production.auth react-scripts build",
```

Comment:
why build:production.auth was removed?

Reply:
I removed it because a think that with one build script should be enough. If a user wants to change the value of a feature flag, they can go an change it in the [env file](https://github.com/label-sleuth/label-sleuth/blob/main/frontend/.env.production). If not, we should have a script and a different env file for each environment variable/feature flag combination.

I will cover this as part of the work of #187